### DISCUSSION TITLE
Telescope Problem

### DISCUSSION THREAD

**Discussion by mjamilasfihani** [tenancy/tenancy#288] [Q&A]
any ideas how to reduce the Telescope watchers? sorry I've no clue on it, or maybe I miss something. thank you

**Comment by ArlonAntonius** [comment]
@mjamilasfihani I don't think I understand your question, it looks like a question that is related more to the configuration of Laravel Telescope itself. Could you elaborate further?

**Comment by mjamilasfihani** [comment]
@ArlonAntonius let's write it down slowly.. 😄 

I have a tenant based on a primary domain:
example.net -> this is the main resource where my Laravel should be placed
example.com -> this is the main tenant (should be defined so everything is worked as well)

the problem is started from here. since I set my Telescope URL as example.net, the HTTP identifier catch everything right? (query, events, and many more..)

so yesterday, I've tried to put an extra config by name is `app.enterprise_url` where belongs to example.com and give a condition inside a HTTP identifier, from there it reduce most of the craziness.

```php
return (config('app.url') == $request->getSchemeAndHttpHost())
    ? null // it can reduce the Telescope except the Events
    : static::query()->where('hostname', $request->getHost())->first();
```

I realized, that Telescope's service is started inside the provider. so, what is the best solution to reduce the Event's catcher?

**Reply by mjamilasfihani** [reply]
let's mark this question as completed, I realize that Events should be catched. Because I only need the Exception and few features to be tracked and it work, the Tenancy also handled the logger nicely.

**Reply by mjamilasfihani** [reply]
actually someone asked earlier why I don't use path inside my Telescope? the nicest answer is "it will continue the craziness"
